client/my-sites/stats/activity-log-tasklist/update.jsx

```diff
@@ -55,9 +65,9 @@ class ActivityLogTaskUpdate extends Component {
 					<div>
 						<span className="activity-log-tasklist__update-text">
 							{ linked ? (
-								<Button borderless onClick={ this.handleNameClick }>
+								<a className={ { borderless: true } } href={ url } onClick={ this.handleNameClick }>
 									{ name }
```

Comment:
can it just be `className="borderless"`

Reply:
i think it needs to be either
`className="borderless"`

or 

`className={ classNames( { borderless: true } ) }`

with your approach you end up with:

`<a class="[object Object]" href="/theme/twentyfifteen/jetpackpitcrew.wpengine.com">Twenty Fifteen</a>`

Reply:
Nice catch. I am just going to remove the class since it wasn't doing anything. We can always add it later. I think it was only there because it used to be a button before.